modules/ags/src/main/scala/lucuma/ags/AgsGuideQuality.scala

```diff
@@ -7,23 +7,29 @@ import cats.Order
 import lucuma.core.util.Enumerated
 
 sealed trait AgsGuideQuality extends Product with Serializable {
+  def tag: String
```

Comment:
Nitpicking: can we make this parameters? Just to avoid the `override val` in all subtypes.

Reply:
ah traits can have params now...

Reply:
Actually we could convert these to enums